.github/ISSUE_TEMPLATE/meeting-sprint-planning.md

```diff
@@ -11,31 +11,29 @@ This is a **60 minute** recurring meeting **every other Wednesday at 7:30AM Paci
 
 Our goal is to synchronize the team on the most important things to work on, and to divide work between one another so we know what to work on next.
 
-- [**Sprint Board**](https://github.com/orgs/2i2c-org/projects/21)
-- [**Deliverables backlog**](https://github.com/orgs/2i2c-org/projects/22)
-
-### Meeting roles
-
+- [**Team backlog**](https://github.com/orgs/2i2c-org/projects/22/views/10)
+  - [list of items with people assigned to them](https://github.com/orgs/2i2c-org/projects/22/views/10)
+  - [list of items without any people assigned](https://github.com/orgs/2i2c-org/projects/22/views/15)
 - **Meeting facilitator**: {{ NAME }}
 
 ### Meeting agenda
 
-_Follow the agenda below, checking boxes as we complete each step._
+**Review team projects.** (30 min)
+
+For each project, its champion does the following:
 
-**Review the Sprint Board.** (10m)
+- Share accomplishments since the last sprint, demos, etc
+- Discuss any quick blockers or major questions to answer
+- Discuss the work plan for the next sprint for this project
 
-- [ ] Celebrate our accomplishments
-- [ ] Archive deliverables we finished
+**Review operations and support items** (20 min)
 
-**Populate Sprint Board with new deliverables** (45m)
+For each operations or support item, we should:
 
-- [ ] Review in-progress deliverables from the last sprint: drop or carry over items as needed
-- [ ] Review a few more urgent items in the [deliverables backlog](https://github.com/orgs/2i2c-org/projects/22/views/1)
-- [ ] Add new deliverables to Sprint Board
-- [ ] Somebody is assigned to each deliverable
-- [ ] Everybody agrees the current cycle's plan is realistic and that the right items are on the board.
+- Assign a team member to items that don't yet have one
```

Comment:
The first option is my interpretation of that sentence.

Reply:
Agree that point 1 is the intended meaning here.

I guess this raises the question of "What if our capacity is maxed out, and we still have support/ops things without an assignee?". I guess in that case we'd need to triage the outstanding operations issues, and prioritize them.

Another point: we don't have to assign operations issues *only* in the team meeting, we can do that asynchronously as well.

Reply:
> Another point: we don't have to assign operations issues only in the team meeting, we can do that asynchronously as well.

Indeed, that is important because operations and support can not wait for a meeting to be assigned if the urgency is high.